tests/unittest.bash

```diff
@@ -0,0 +1,801 @@
+#!/bin/bash
```

Comment:
Should this be in bazel's distro given how many copies of it now seem to exist?

Reply:
Probably, but I don't have the cycles to make this happen right now.